llvm/lib/Target/WebAssembly/AsmParser/WebAssemblyAsmParser.cpp

```diff
@@ -69,12 +69,23 @@ struct WebAssemblyOperand : public MCParsedAsmOperand {
     std::vector<unsigned> List;
   };
 
+  struct CaLOpElem {
+    int64_t Opcode;
+    const MCExpr *Tag;
+    int64_t Dest;
```

Comment:
I made them `int64_t` just because the token's parser's return value was `int64_t`:
https://github.com/llvm/llvm-project/blob/e99755d41c8d5ee8594fa161ac05fbef02d902aa/llvm/include/llvm/MC/MCAsmMacro.h#L115-L118

And yeah I agree giving them the real type is better. Changed `Opcode` to `uint8_t` and `Dest` to `unsigned` to reflect the real binary spec.